letsencrypt/build.yaml

```
@@ -12,6 +12,7 @@ args:
   CRYPTOGRAPHY_VERSION: 36.0.2
   CERTBOT_DNS_AZURE_VERSION: 1.5.0
   CERTBOT_DNS_DIRECTADMIN_VERSION: 0.0.24
+  CERTBOT_DNS_DUCKDNS_VERSION: v1.3
```

It seems that `v` isn't hurting here, but for consistency I'll drop it.

```suggestion
  CERTBOT_DNS_DUCKDNS_VERSION: 1.3
```